pipeline.yml

```diff
@@ -10,7 +10,7 @@ env:
   - DOCKER_CONTAINER : "buildContainer"
   - HARMONY_REPO : "DynamoDS/PackageManagerClient"
   
-check_changelog_updated_on_pr: false
```

Comment:
oh so this is a supported step OOTB from the CI lib?

Reply:
thats what the docs say...